main.go

```diff
@@ -58,8 +46,15 @@ func proxy(ctx context.Context, config *Config, signals chan os.Signal) error {
 		if !errdefs.IsNotFound(err) {
 			return err
 		}
+		image, err := config.GetImage(ctx, client)
+		if err != nil {
+			return err
+		}
 		// create new container
-		if container, err = create(ctx, client, config); err != nil {
+		if container, err = client.NewContainer(ctx, config.ID,
+			WithCurrentSpec(config),
```

Comment:
Does `Scope` automatically get handled here?

Reply:
no, i'm not sure we need anything specific to scope

Reply:
Or sorry, I meant does this handle setting the label for scope?

Reply:
Since we'll need to set initial scope on container creation right?

Reply:
ahh, let me add that